backend/.env.template

```diff
@@ -66,4 +66,7 @@ FEDERATION_XCOM_SENDCOINS_ENABLED=$FEDERATION_XCOM_SENDCOINS_ENABLED
 
 # GMS
 GMS_ACTIVE=$GMS_ACTIVE
-GMS_URL=$GMS_URL
+GMS_API_URL=$GMS_API_URL
+GMS_DASHBOARD_URL=$GMS_DASHBOARD_URL
+GMS_WEBHOOK_SECRET=$GMS_WEBHOOK_SECRET
+GMS_CREATE_USER_THROW_ERRORS=$GMS_CREATE_USER_THROW_ERRORS
```

Comment:
please add to deployment/bare_metal/.env.dist

Reply:
done